examples/gym/run-example.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
`run-atari.sh`?  would be more in line with other script names, as they are all examples

Reply:
or even better, `run-breakout.sh`